### DISCUSSION TITLE
[Announcement] lightening talks from the top scorers

### DISCUSSION THREAD

**Discussion by KihongHeo** [prosyslab-classroom/cs424-program-reasoning#357] [Announcements]
Hi.

I am amazed that you folks have very high scores in HW4. Really good job everybody.
So, I would like to ask the top 3 scorers to give short presentations about their strategies.

@Lee-Janggun , @askme143 , @PSuHyeon , can you present next Tue? up to 5 min?
Please send your slides to me before the class.

**Comment by Lee-Janggun** [comment]
I will be glad to give a presentation. Should it be in English or Korean? I am assuming in English as the class typically is.

**Comment by KihongHeo** [comment]
English.

**Comment by Lee-Janggun** [comment]
Thanks for the confirmation. I'll send the slides by Monday.

**Comment by PSuHyeon** [comment]
Thank you for providing such a good experience. I will prepare slides for presentation by Tues.

**Comment by askme143** [comment]
Thank you! I'll send slides by Monday.
